parquet-protobuf/pom.xml

```diff
@@ -57,6 +58,16 @@
       <artifactId>protobuf-java</artifactId>
       <version>${protobuf.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java-util</artifactId>
+      <version>${protobuf.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>proto-google-common-protos</artifactId>
```

Comment:
why this dependency, i thought well known types shipped with the proto package?

Reply:
If you want to pre-packaged common Proto classes, you need to include this.